extensions/jwt-decoder/src/jwt-view.tsx

```diff
@@ -38,7 +38,7 @@ const JwtView = () => {
     const headItems = ListFromObject(header, claims);
     const dataItems = ListFromObject(data, claims);
 
-    const tokenImg = `<img alt="view token" width="720" src="data:image/svg+xml,${encodeURI(
+    const tokenImg = `<img alt="view token" width="720" src="data:image/svg+xml,${encodeURIComponent(
```

Comment:
There's another use of encodeURI in `jwt-list.tsx` can you replace that one too?

Reply:
I have replaced the encodeURI in `jwt-list.tsx` as suggested. Thank you.